src/servicesHost.ts

```diff
@@ -670,9 +749,7 @@ function addCache(servicesHost: typescript.ModuleResolutionHost) {
       const cache = createCache<ReturnType<typeof originalFunction>>(
         originalFunction
       );
-      servicesHost[
-        functionToCache
-      ] = cache.getCached as typescript.ModuleResolutionHost[CacheableFunction];
+      servicesHost[functionToCache] = cache.getCached as any;
```

Comment:
could we fix the `any` please?

Reply:
See also https://github.com/TypeStrong/ts-loader/pull/954#discussion_r294036783 where @fa93hws has bumped on this too.